tutorials/introductory/usage.py

```diff
@@ -657,7 +657,7 @@ def my_plotter(ax, data1, data2, param_dict):
 # :rc:`path.simplify` and :rc:`path.simplify_threshold`, which
 # can be defined e.g. in the :file:`matplotlibrc` file (see
 # :doc:`/tutorials/introductory/customizing` for more information about
-# the :file:`matplotlibrc` file). :rc:`path.simplify` is a boolean
+# the :file:`matplotlibrc` file). :rc:`path.simplify` is a Boolean
```

Comment:
I think boolean is lowercase?

Reply:
Found some inconsistencies in the Python docs, [Build-in Types](https://docs.python.org/3/library/stdtypes.html#boolean-operations-and-or-not) use the capital "B" and [Boolean Objects](https://docs.python.org/3/c-api/bool.html?highlight=boolean) does not.